assets/js/components/notifications/BannerNotification/index.js

```diff
@@ -153,11 +157,14 @@ function BannerNotification( {
 	async function handleCTAClick( e ) {
 		e.persist();
 
+		let dismissOnCTAClick = true;
 		if ( onCTAClick ) {
-			await onCTAClick( e );
+			setIsAwaitingCTAResponse( true );
+			( { dismissOnCTAClick } = ( await onCTAClick( e ) ) || {} );
```

Comment:
This will break dismissible CTAs that don't return this new property because the declared `dismissOnCTAClick` default above will be overridden with `undefined` here.

We need to be sure the default `true` is preserved even when it's not returned as proposed in the IB.

Reply:
Yikes - was trying something and removed the default.